src/System.Runtime/src/System.Runtime.Forwards.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license. 
+// See the LICENSE file in the project root for more information. 
+
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Runtime.InteropServices.GCHandle))]
```

Comment:
I asked @botaberg to add these to fix the tests which were compiling against old packages and running on live bits.  Yeah, we should remove them before we ship.